src/default-settings.js

```diff
@@ -1,167 +1,177 @@
-            "type":"circle",
-            "attributes":{'stroke-width': 3, 'stroke': 'black'},
-            "radius":5,
-            "values":{"AESER": ["Yes", 'Y']}
-        }
-    ],
-    "legend":{
-        "mark":"circle", 
-        "label": 'Severity'
-    },
-    "margin": {"top": 50, bottom: null, left: null, right: null},
-    "colors": ['#66bd63', '#fdae61', '#d73027', '#6e016b'],
-    "date_format":"%m/%d/%y",
-    "resizable":true,
-    "max_width":1000,
-    "y_behavior": 'flex',
-    "gridlines":"y",
-    "no_text_size":false,
-    "range_band":15,
-    "color_by":null //set in syncSettings()
+import clone
+    from './util/clone';
+
+const settings =
+
+  //Template-specific settings
+    {stdy_col: 'ASTDY'
+    ,endy_col: 'AENDY'
+    ,id_col: 'USUBJID'
+    ,rfendt_col: 'RFENDTC'
+    ,seq_col: 'AESEQ'
+    ,sev_col: 'AESEV'
+    ,sev_vals:
+        ['MILD'
+        ,'MODERATE'
+        ,'SEVERE']
+    ,ser_col: 'AESER'
+    ,term_col: 'AETERM'
+    ,filter_cols: []
+    ,detail_cols: []
+
+  //Standard chart settings
+    ,x: {column: 'wc_value'
+        ,type: 'linear'
+        ,label: null}
+    ,y: {column: null // set in syncSettings()
+        ,type: 'ordinal'
+        ,label: ''
+        ,sort: 'earliest'
+        ,behavior: 'flex'}
+    ,marks: 
+        [
+            {type: 'line'
+            ,per: null // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'stroke-width': 5
+                ,'stroke-opacity': .5}}
+        ,
+            {type: 'circle'
+            ,per: null // set in syncSettings()
+            ,tooltip: null
+            ,attributes:
+                {'fill-opacity': .5
+                ,'stroke-opacity': .5}} // set in syncSettings()
+        ,
+            {type: 'line'
+            ,per: null // set in syncSettings()
+            ,values: {} // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'class': 'serious'
+                ,'stroke': 'black'
+                ,'stroke-width': 2}}
+        ,
+            {type: 'circle'
+            ,per: null // set in syncSettings()
+            ,values: {} // set in syncSettings()
+            ,tooltip: null // set in syncSettings()
+            ,attributes:
+                {'class': 'serious'
+                ,'fill': 'none'
+                ,'stroke': 'black'
+                ,'stroke-width': 2}}
+        ]
+    ,legend:
+        {location: 'top'
+        ,label: 'Severity'}
+    ,color_by: null // set in syncSettings()
+    ,colors:
+        ['#66bd63'
+        ,'#fdae61'
+        ,'#d73027']
+    ,date_format: '%Y-%m-%d'
+    ,y_behavior: 'flex'
+    ,gridlines: 'y'
+    ,no_text_size: false
+    ,range_band: 15
+    ,margin: {top: 50}
+    ,resizable: true
 };
 
-export function syncSettings(preSettings){
+export function syncSettings(preSettings) {
     const nextSettings = Object.create(preSettings);
+
```

Comment:
So if the user doesn't provide settings we use the defaults ... seems reasonable ...